client/environment_drift_detection.go

```diff
@@ -0,0 +1,31 @@
+package client
+
+func (self *ApiClient) EnvironmentDriftDetection(environmentId string) (EnvironmentSchedulingExpression, error) {
+	var result EnvironmentSchedulingExpression
+
+	err := self.http.Get("/scheduling/drift-detection/environments/"+environmentId, nil, &result)
+	if err != nil {
+		return result, err
+	}
+	return result, nil
+}
+
+func (self *ApiClient) EnvironmentUpdateDriftDetection(environmentId string, payload EnvironmentSchedulingExpression) (EnvironmentSchedulingExpression, error) {
+	var result EnvironmentSchedulingExpression
+
+	err := self.http.Put("/scheduling/drift-detection/environments/"+environmentId, payload, &result)
+	if err != nil {
+		return EnvironmentSchedulingExpression{}, err
+	}
+
+	return result, nil
+}
+
+func (self *ApiClient) EnvironmentStopDriftDetection(environmentId string) error {
+	err := self.http.Put("/scheduling/drift-detection/environments/"+environmentId, EnvironmentSchedulingExpression{Enabled: false, Cron: ""}, &EnvironmentScheduling{})
```

Comment:
Sorry @eranelbaz but another thing that was just raised. `cron` would now be optional, and `''` wouldn't really work well in the API, as it's a patch

https://env0.slack.com/archives/C02TRUU5YS0/p1646659554487259